lib/puppet/type/ntp_config.rb

```diff
@@ -15,6 +15,11 @@
     end
   end
 
+  newproperty(:authenticate) do
```

Comment:
The type defs throughout are expecting symbols for :true/:false, presumably to support puppet < 3.4  Downstream providers (Cisco and Arista) are expecting this behavior.  I propose we defer this change, and consider it only if we opt to refactor the module.